module/utils.mjs

```diff
@@ -492,8 +492,8 @@ export function getTargetDescriptors() {
  * @returns {Token5e[]}
  */
 export function getSceneTargets() {
-  let targets = canvas.tokens.controlled.filter(t => t.actor);
-  if ( !targets.length && game.user.character ) targets = game.user.character.getActiveTokens();
+  let targets = canvas.tokens?.controlled?.filter(t => t.actor);
+  if ( !targets?.length && game.user.character ) targets = game.user.character.getActiveTokens();
```

Comment:
This might result in this function not returning an array if the current user doesn't have an assigned token, so we should tweak this to ensure an array is always returned:

```suggestion
  let targets = canvas.tokens?.controlled.filter(t => t.actor) ?? [];
  if ( !targets.length && game.user.character ) targets = game.user.character.getActiveTokens();
```

Was there a situation where `canvas.tokens` was defined but `canvas.tokens.controlled` was `undefined`? If not, then we can ignore the second question mark.

Reply:
Nah, just sloppy on my part. Suggestions pushed!